sdf.md

```diff
@@ -1399,13 +1405,20 @@ The requirements for high level composition include the following:
 
 ## Paths in the model namespaces
 
-The model namespace is organized according to terms that are defined in the definition files that are present in the namespace. For example, definitions that originate from an organization or vendor are expected to be in a namespace that is specific to that organization or vendor. There is expected to be an SDF namespace for common SDF definitions used in OneDM.
+The model namespace is organized according to terms that are defined
+in the SDF documents that are present in the namespace. For example, definitions that originate from an organization or vendor are expected to be in a namespace that is specific to that organization or vendor. There is expected to be an SDF namespace for common SDF definitions used in OneDM.
```

Comment:
Not the main point of this PR, but besides in this sentence, OneDM is only mentioned in an example and the abbreviated title. Should the term be explained here? Or the sentence removed?

Reply:
Indeed, probably the OneDM note is not needed here.

Reply:
Also now realised "that are present in the namespace" is a bit confusing here. Would something like "that contribute to the namespace" be more clear?